fastlane_bot/events/async_event_update_utils.py

```diff
@@ -12,11 +14,12 @@
 
 from fastlane_bot.data.abi import ERC20_ABI
 from fastlane_bot.events.async_utils import get_contract_chunks
-from fastlane_bot.events.exchanges import exchange_factory
+from fastlane_bot.events.exceptions import AyncUpdateRetryException
```

Comment:
`AyncUpdateRetryException` seems to be missing an "s"

Reply:
this is a custom implementation